pkg/controller/stacks/stack/stack.go

```diff
@@ -722,6 +726,9 @@ func (h *stackHandler) prepareHostAwareJob(j *batch.Job, tokenSecret string) err
 	j.Name = o.Name
 	j.Namespace = o.Namespace
 
+	a := hosted.ObjectReferenceAnnotationsOnHost("stack", h.ext.GetName(), h.ext.GetNamespace())
```

Comment:
i think the function this is inside of, `prepareHostAwareJob` is a missed remnant of the old job support we used to have and we recently removed.  I think this entire function can be removed.  It's not called from anywhere in the codebase except for tests.

Reply:
This brings me great joy.